Code/Tools/FBuild/FBuildCore/Graph/CSNode.h

```diff
@@ -31,7 +31,7 @@ class CSNode : public FileNode
     virtual bool DoDynamicDependencies( NodeGraph & nodeGraph, bool forceClean ) override;
     virtual BuildResult DoBuild( Job * job ) override;
 
-    inline Node * GetCompiler() const { return m_StaticDependencies[ 0 ].GetNode(); }
+	CompilerNode * GetCompiler() const;
```

Comment:
GetCompiler functions were changed to return CompilerNodes and implementation moved to the cpp to keep includes low.